se2_planning_ros/src/OmplReedsSheppPlannerRos.cpp

```diff
@@ -73,10 +138,66 @@ void OmplReedsSheppPlannerRos::publishPath() const {
   planner_->getPath(&rsPath);
   se2_navigation_msgs::Path msg = se2_planning::convert(rsPath);
   msg.header_.frame_id = parameters_.pathFrame_;
-  msg.header_.stamp = ros::Time::now();
+  msg.header_.stamp = planTimeStamp_;
   msg.header_.seq = planSeqNumber_;
   pathPublisher_.publish(se2_navigation_msgs::convert(msg));
   ROS_INFO_STREAM("Publishing ReedsShepp path, num states: " << rsPath.numPoints());
 }
 
+void OmplReedsSheppPlannerRos::publishStartGoalMsgs(const ReedsSheppState& start, const ReedsSheppState& goal) const {
+  geometry_msgs::PoseStamped startPose;
+  startPose.header.frame_id = parameters_.pathFrame_;
+  startPose.header.stamp = planTimeStamp_;
+  startPose.pose = se2_planning::convert(start);
+  startPublisher_.publish(startPose);
+  geometry_msgs::PoseStamped goalPose;
+  goalPose.header.frame_id = parameters_.pathFrame_;
+  goalPose.header.stamp = planTimeStamp_;
+  goalPose.pose = se2_planning::convert(goal);
+  goalPublisher_.publish(goalPose);
+}
+
+void OmplReedsSheppPlannerRos::initializeStateSpaceMarker() {
```

Comment:
consider renaming to state space Boundary marker. that is what they are doing, right?

Reply:
Good point!

Reply:
Done in 0d0bcc061f20563fb01b03194bbd5181cd9bb672.